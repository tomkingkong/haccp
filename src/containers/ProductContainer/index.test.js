import React from 'react';
import { shallow } from 'enzyme';

import { ProductContainer } from '.';

describe('ProductContainer', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<ProductContainer userProducts={[]} />);
  });

  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});