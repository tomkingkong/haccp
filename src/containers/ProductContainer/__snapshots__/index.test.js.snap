// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductContainer should match snapshot 1`] = `
<section
  className="products-container"
>
  <WithStyles(Button)
    onClick={[Function]}
    variant="outlined"
  >
    <pure(AddIcon) />
  </WithStyles(Button)>
</section>
`;
